test/distributed/org/apache/cassandra/distributed/shared/ClusterUtils.java

```diff
@@ -572,25 +574,44 @@ private static class ClusterMetadataVersion
     {
         public final int node;
         public final Epoch epoch;
-        public final Epoch replicated;
```

Comment:
@ifesdjeen I couldn't figure out why this was present... its the same as `epoch` and only used in one code path... so removing made the error string easier to read... I can revert if you want